fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/DerbyBootstrapper.java

```diff
@@ -197,31 +199,48 @@ public static void bootstrapOauthDb(DataSource ds) throws Exception {
      * @throws SQLException
      */
     public static void bootstrapBatchDb(DataSource ds) throws SQLException {
+        // This is specific to boostrapping where we are not suppose to use the derby db, rather a remote db.
+        boolean isDerby = false;
         try (Connection c = ds.getConnection()) {
-            try {
-                JdbcTarget target = new JdbcTarget(c);
-                IDatabaseAdapter adapter = new DerbyAdapter(target);
-
-                // Set up the version history service first if it doesn't yet exist
-                CreateVersionHistory.createTableIfNeeded(ADMIN_SCHEMANAME, adapter);
-
-                // Current version history for the database. This is used by applyWithHistory
-                // to determine which updates to apply and to record the new changes as they
-                // are applied
-                VersionHistoryService vhs =
-                        new VersionHistoryService(ADMIN_SCHEMANAME, BATCH_SCHEMANAME);
-                vhs.setTarget(adapter);
-                vhs.init();
+            try (Statement stmt = c.createStatement();
```

Comment:
I thought you may want to check the datasource configuration in server.xml, but this also works. thanks!

Reply:
yes, that was one approach, I considered